src/app/question/list-entry/question-list-entry.component.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { QuestionListEntryComponent } from './question-list-entry.component';
+import { RouterTestingModule } from '@angular/router/testing';
+import { By } from '@angular/platform-browser';
+import { Component, OnInit, DebugElement } from '@angular/core';
+import { FlagComponent } from '../../profile/flag/flag.component';
+import { MaterialModule } from '@angular/material';
+import { MomentModule } from 'angular2-moment';
+import { LinkyModule } from 'angular-linky';
+import { AvatarDirective } from '../../profile/avatar.directive';
+import { EmojiModule } from '../../emoji/emoji.module';
+import { MeditatedRecentlyDirective } from '../../profile/meditated-recently.directive';
+import { QuestionService } from '../question.service';
+import { FakeQuestionService } from '../testing/fake-question.service';
+
+@Component({
+  template: `
+    <question-list-entry [question]="question" [mode]="mode" [isAdmin]="isAdmin">
+    </question-list-entry>`
+})
+
+
+class TestHostComponent implements OnInit {
+  question: any;
+  isAdmin: boolean;
+  mode: number;
+
+  public ngOnInit() {
+    this.isAdmin = false;
+    this.mode = 0;
+
+    this.question = {
+      '_id': '5951b4b7800d4a3a760c1426',
```

Comment:
If this is an id from the website, please change. (also for username, gravatarHash)